### ISSUE TITLE
how much epoch I need to get the realistic results?

### ISSUE THREAD

**Issue by elsobhano** [junyanz/pytorch-CycleGAN-and-pix2pix#1391]
Hi everybody 

I trained my network for 30 epochs. Is it normal that I don't have good results?

**Comment by tarmopungas** [comment]
It could be the case. If possible, just try running for more epochs and compare results. Also, you can use checkpoints to verify that the model is still improving.

**Comment by zihaog0724** [comment]
It depends on your data. Intuitively, however, >100 epoch is more likely to give promising result. I've tried pix2pix on my own dataset and it starts to generate good result after 100 epoch. Also you could try learning rate decay.

**Comment by Sarmadfismael** [comment]
Hi all 
my training take long time to get the result 1 hour per epoch, is it normal ? 
note: the input image size 512x512
can i stop the training in any step i want??
